### ISSUE TITLE
Link Twitter Account button errors out

### ISSUE THREAD

**Issue by cuzimabot** [thepracticaldev/dev.to#7585]
**Describe the bug**
Unable to link account to twitter

**To Reproduce**
Go to settings > Click in Link Twitter account

**Expected behavior**
Authorize Twitter and link accounts

**Screenshots**
![image](https://user-images.githubusercontent.com/11393226/80561160-8d3ca500-89b1-11ea-9d7a-4917886ce929.png)

- OS: Windows 10 
- Browser: Chrome
- Version: Version 81.0.4044.129 (Official Build) (64-bit)

**Comment by citizen428** [comment]
Thanks for the issue @cuzimabot! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by citizen428** [comment]
@rhymes Tagging you directly since you just worked on this. 

I tried this in a private browsing session and could NOT reproduce it. When I then removed Twitter auth from my main account and I tried to re-add it, I experience the issue described here.

**Comment by rhymes** [comment]
I tried both with Twitter and with GitHub locally on master (which at the present moment contains the same code that production does), both from the "connect twitter accont" button in the account and with one of the global sign in buttons but to no avail.

I then tried the same thing in production and was able to reproduce the problem, which only happens with the "Connect button", I'm going to investigate.

Thank you both!
